src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -43,37 +45,43 @@ export default function useGetSearchResults(articles, query, page) {
           );
         });
       });
+      // =====
+      // End of filtering logic.
```

Comment:
Would it be possible to break the sets of logic into functions instead of needing comments to differentiate? Like `filterSearchResults` and then `orderSearchResults`? Or maybe it is too messy being in a React hook

Reply:
Ya that was my fear too, separating them into different functions could help but it also might make it so that over time we have to jump around a lot to find out what that hook is doing

Reply:
Yeah it might be better the way you have it already then

Reply:
If it gets much larger though I think it'll be a good opportunity for abstraction, hopefully we move to a dedicated 3rd party search service by then though